rama-net/src/address/authority.rs

```diff
@@ -229,43 +218,47 @@ impl<'de> serde::Deserialize<'de> for Authority {
 mod tests {
     use super::*;
 
-    fn assert_eq(s: &str, authority: Authority, host: &str, port: u16) {
```

Comment:
So sorry! I totally forgot and didn't read that I had defined a custom `assert_eq` function... Can you please revert this test. E.g. leave `test_parse_valid` untouched and put this code back.

Also please do a similar thing for your `socket_address.rs`' `test_parse_valid` test. This old approach is much cleaner.

So sorry. I had read it wrongly as `assert_eq!`. But now that I see you deleting `assert_eq` (a custom fn I had defined) I now realise I did not make a mistake after all.

You did a lot already though, and I don't want to put you into a ping-pong show. So do tell me if this is too much and I'll take it over for this minor change. But if you are up for it please do so, as this is probably the last bit before we can merge.

Reply:
Thanks for clarifying! No problem, I’ll revert the `test_parse_valid` changes in both `authority.rs` and `socket_address.rs` to restore the use of the custom`assert_eq` function.

I’ll update the PR shortly!